### DISCUSSION TITLE
HTML WYSIWYG and images

### DISCUSSION THREAD

**Discussion by kevinsmgov** [OrchardCMS/OrchardCore#10144] [Q&A]
Is anyone aware of a way to get the HTML editor to embed image tags instead of short codes (the way it used to work)?

**Comment by Skrypt** [comment]
Is there a specific reason why you would need this?
Also which editor are you refering to? Monaco, Trumbowyg, Markdown?

**Reply by Skrypt** [reply]
Here the thing is that using a shortcode has some advantages. If you embed an image tag, the path to your file is fixed while the image shortcode will resolve the proper path based on the tenant where it stands. If there is something that is missing on the image shortcode then you/we might want to add what's missing instead.

**Reply by kevinsmgov** [reply]
We just migrated from Orchard 1 to Orchard Core. Our content authors are used to seeing the images in the wysiwyg editor and the new approach is getting complaints. Previous content from the migrated system has the images visible (and the new ability to resize them inline) but they can't update images in content items the way they are used to.

**Reply by kevinsmgov** [reply]
I guess the editor is Trumbowyg (it's not Monaco or Markdown)

**Reply by Skrypt** [reply]
Then we might simply need some javascript added to the editors to render these images as they should.

**Reply by kevinsmgov** [reply]
I believe the change happened in \src\OrchardCore.Modules\OrchardCore.ContentFields\wwwroot\Scripts\trumbowyg.media.tag.js (commit 99cfa8e6 on 5/19/2020)

I'm curious if there is a way for me to overwrite that script on top of the current NuGet packages that my project uses (I'd rather not resort to building my site on top of the full Orchard source code)

**Reply by deanmarcussen** [reply]
It really didn't. That pr was just moving from inserting liquid, to inserting shortcodes

The reason we need to insert something, i.e. liquid or shortcodes, is to be able to resolve the correct tenant prefix for the media.
otherwise it would not be a thing

**Reply by kevinsmgov** [reply]
I see. Thanks for the information.
